Dockerfile

```diff
@@ -24,10 +24,14 @@ ENV CONTAINER_REPOSITORY_CONTEXT winery
 ENV CONTAINER_BUS_MANAGEMENT_MOCK false
 ENV CONTAINER_DEPLOYMENT_TESTS false
 
-ENV ENGINE_IA_HOSTNAME localhost
-ENV ENGINE_IA_PORT 8090
-ENV ENGINE_IA_USER admin
-ENV ENGINE_IA_PASSWORD admin
+ENV ENGINE_IA_JAVA8_HOSTNAME localhost
```

Comment:
I wouldn't change the existing ones to avoid breaking the current docker-compose setup